react/Logo/Logo.js

```diff
@@ -1,31 +1,66 @@
 import styles from './Logo.less';
-import logoMarkup from './logo.svg';
+import LogoText from './LogoText';
+import LogoIcon from './LogoIcon';
 
 import React from 'react';
 import PropTypes from 'prop-types';
 
 import classnames from 'classnames';
 
-export default function Logo({ svgClassName, invert, ...restProps }) {
+const viewportDimensions = type => ({
+  default: { height: 68.031, width: 170.079 },
+  compact: { height: 68.031, width: 68.031 }
+})[type];
+
+const viewPortDimensionPointer = type => dimension => viewportDimensions(type)[dimension];
```

Comment:
Remove currying and change to a 2 arity function `[type, dimension]` so that the following construction is possible:

```javascript
const height = getViewportDimension(type, 'height')
```